frontend/integration-tests/tests/cluster-settings.scenario.ts

```diff
@@ -0,0 +1,220 @@
+import { browser} from 'protractor';
+import * as _ from 'lodash';
+import { safeDump, safeLoad } from 'js-yaml';
+
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+import * as clusterSettingsView from '../views/cluster-settings.view';
+import * as horizontalnavView from '../views/horizontal-nav.view';
+import * as yamlView from '../views/yaml.view';
+
+
+describe('Cluster Settings', () => {
+  const labels = {app: `${testName}-app`};
+  const name = `${testName}-cell`;
+  const text = `${name} Logs`;
+  beforeEach(async() => {
+    await browser.get(`${appHost}/settings/cluster/${testName}`);
+    await crudView.isLoaded();
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('display page title, horizontal navigation tab headings and pages', async() => {
+    expect(clusterSettingsView.heading.isPresent()).toBe(true);
+    await horizontalnavView.clickHorizontalTab('Overview');
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('Cluster Operators');
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+  });
+  it('display overview channel update modal, update value and save, and check if selected value equal to updated value', async() => {
+    await horizontalnavView.clickHorizontalTab('Overview');
+    await crudView.isLoaded();
+    expect(clusterSettingsView.channelUpdateLink.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelUpdateLink.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.channelDropdownButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelDropdownButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.getSelectedChannel.get(0).isDisplayed()).toBe(true);
+    const selectedChannelValue = clusterSettingsView.getSelectedChannel.get(0).getText();
+    await clusterSettingsView.getSelectedChannel.get(0).click();
+
+    expect(clusterSettingsView.confirmActionButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.confirmActionButton.click();
```

Comment:
A little scary changing the `ClusterVersion` values in our integration tests. We might skip clicking save here. If we do save, we probably want to change it back to the original value.